src/classes/keyframe_scaler.py

```diff
@@ -0,0 +1,87 @@
+"""
+ @file
+ @brief Process project data, scaling keyframe X coordinates by the given factor
+ @author Noah Figg <eggmunkee@hotmail.com>
```

Comment:
I think only Myself and you need to be listed here

Reply:
Fixed.